pom.xml

```diff
@@ -33,6 +33,10 @@
             <id>scijava.public</id>
             <url>https://maven.scijava.org/content/groups/public</url>
         </repository>
+        <repository>
```

Comment:
what is the jitpack dependency for?

Reply:
I have no idea, I'm just following mcib3d recommendation from the [github page ](https://github.com/mcib3d/mcib3d-core)

Reply:
👍